assets/parquet/Dockerfile

```
@@ -0,0 +1,68 @@
+#     tmux \
+#     vim \
+#     libsnappy-dev \
+#     libdouble-conversion-dev \
+#     libre2-dev \
+#     libgflags-dev \
+#     libboost-all-dev;
+
+# Install Task
+RUN cd /usr/local && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d;
+
+RUN pip3 install pyarrow;
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)